docs/framework/migration-guide/retargeting/foreach_iterator_variable_is_now_scoped_within_the_iteration_so_closure_capturing_semantics_are_different_in_c5.md

```diff
@@ -0,0 +1,10 @@
+### Foreach iterator variable is now scoped within the iteration, so closure capturing semantics are different (in C\#5)
+
+|   |   |
+|---|---|
+|Details|Beginning with C# 5 (Visual Studio 2012), foreach iterator variables are scoped within the iteration. This can cause breaks if code was previously depending on the variables to not be included in the foreach&#39;s closure. The symptom of this change will be that an iterator variable passed to a delagate will be treated as the value it had at the time the delegate was created, rather than the value it had at the time the delegate was invoked.|
```

Comment:
inline code foreach?
will be -> try to use present tense as much as possible. it improves readability
you don't have to encode apostrophes
typo: delagate